UHF/simulated_uhf.py

```diff
@@ -181,25 +190,32 @@ def receive_msg(client, buffer, lock):
         try:
             # Use select to check if data is available to be read
             ready_to_read, _, _ = select.select([client], [], [], 0.5)  # timeout set to 0.5 seconds
+
             if ready_to_read:
                 message = client.recv(BUFF_SIZE)
+
                 if message:
                     if DEBUG:
                         print(f"Received: {message.decode('utf-8')}")
-                    with lock:
-                        buffer.append(message)
+
+                    _, port = client.getsockname()
+
+                    # Primitive check to see if messages intent was to modify UHF params
+                    if port == COMMS_SIDE_SERVER_PORT and b"MOD_UHF" in message:
```

Comment:
Good work. Could you please add a README or a comment at the top explaining how to update the beacon. I know its simple now, but as we add functionality things will get buried and I would like to keep it immediately clear for other members.